python/mlcroissant/mlcroissant/_src/core/json_ld.py

```diff
@@ -102,6 +106,14 @@ def remove_empty_values(d: Json) -> Json:
     return {k: v for k, v in d.items() if v}
 
 
+def formatted_uid_to_json(ctx: Context, uuid):
```

Comment:
To be put in uuid.py?

Reply:
Right, who knows why I didn't in the first place. Done.